BUILD

```diff
@@ -19,3 +19,5 @@ lines_sorted_test(
     error_message = "Authors must be sorted by first name",
     file = "AUTHORS",
 )
+
+load("@io_bazel_rules_go//go:def.bzl", "go_prefix")
```

Comment:
Oops, that shouldn't have made it into this commit.

I think this is left over from when I ran without setting -go_rules_bzl_only_for_internal_use. Maybe gazelle should detect the repository name (and whether it's running in this repository) automatically.